account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/core/TekManager.java

```diff
@@ -82,6 +70,22 @@ public HpkePublicKey getPublicKey() throws GeneralSecurityException, InvalidProt
         return toHpkePublicKey(getHpkeParams(), ksHandler);
     }
 
+    public String publicKey() throws GeneralSecurityException, InvalidProtocolBufferException {
+        KeysetHandle ksHandler = this.getKeysetHandle();
+        if (ksHandler == null) {
+            return null;
+        }
+        return org.bouncycastle.util.encoders.Hex.toHexString(getPublicKey().getPublicKeyBytes().toByteArray());
+    }
+
+    public HpkePrivateKey getPrivateKey() throws GeneralSecurityException, InvalidProtocolBufferException {
```

Comment:
(Nit) inconsistent naming between `publicKey()` and `getPrivateKey()`.

Reply:
Done